core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -18,16 +18,40 @@
       GLOBALS.INTERACTION_SPECS = JSON.parse(
         '{{INTERACTION_SPECS|js_string}}');
     </script>
+
   {% endif %}
 
   {{dependencies_html}}
+  <style>
```

Comment:
@seanlip @naveenshukla Any reviews on latest changes? I have added oppia specific class along with bootstrap classes, so it will not effect bootstrap classes anymore. Although i haven't found any other way to limit this behavior other than including inline CSS, insight on this topic will be helpful.